feat: add app_name for http outputs (#2151)
kyma-project/telemetry-manager: docs/user/02-logs.md

  "traceID": "123"
}
```

> ### Further Enrichment

Author: Aligned by talking to each other

Additionally, the agent enriches every log record with the `cluster_identifier` attribute by setting the APIServer URL of the underlying Kubernetes cluster:

```json
{
   "cluster_identifier": "<APIServer URL>"
   ...
}
```

> For LogPipelines that use an HTTP output, additionally, the following attributes are enriched for an optimized user experience when integrating with SAP Cloud Logging:

Reviewer: ```suggestion
For LogPipelines that use an HTTP output, the following attributes are enriched for an optimized user experience when integrating with SAP Cloud Logging:
```
I just noticed we have additional in the first paragraph. Let's skip it here.

> For LogPipelines which are using an HTTP output, additionally the following attributes are enriched for an optimzed user experience when integrating with SAP Cloud Logging:

Reviewer: ```suggestion
For LogPipelines that use an HTTP output, additionally, the following attributes are enriched for an optimized user experience when integrating with SAP Cloud Logging:
```
So they are not new but enriched? Or were added to enrich?

```json
{
  "@timestamp": "<value of attribute 'time'>",
  "date": "<agent time in iso8601>",
  "kubernetes": {
    "app_name": "<value of pod label 'app.kubernetes.io/name' or 'app'>"
    ...
  },
  ...
}
```

The enriched timestamp attributes have the following meaning:

- **time**: The time when the container runtime captured the log on `stdout/stderr`, which is very close to the time when the log originated in the application.
- **date**: The time when the log agent processed the log, which is later than the value in `time`.

> - **@timestamp**: contains the same value as **time** and is present for an optimized experience with the SAP Cloud Logging integration.

Reviewer: ```suggestion
- **@timestamp**: Contains the same value as **time** and is present for an optimized experience with the SAP Cloud Logging integration.
```

> - **time**: the time when the container runtime captured the log on `stdout/stderr`, which is timely very close to the time when the log originated in the application.
> - **date**: the time when the log agent processed the log, which is later then the value in `time`.
> - **@timestamp**: contains the same value as `time` and is present for an optimized experience with the SAP Cloud Logging integration.

Reviewer: ```suggestion
- **time**: The time when the container runtime captured the log on `stdout/stderr`, which is very close to the time when the log originated in the application.
- **date**: The time when the log agent processed the log, which is later than the value in `time`.
- **@timestamp**: contains the same value as **time** and is present for an optimized experience with the SAP Cloud Logging integration.
```
